app/services/idv/steps/doc_auth_base_step.rb

```diff
@@ -26,25 +26,6 @@ def save_proofing_components
         ProofingComponent.create_or_find_by(user: current_user).update(component_attributes)
       end
 
-      # @param [DocAuth::Response,
-      #   DocumentCaptureSessionAsyncResult,
-      #   DocumentCaptureSessionResult] response
-      def extract_pii_from_doc(response, store_in_session: false)
```

Comment:
I thought I deleted this in my last PR, but I must not have checked it in. As far as I can tell this is not used by the in-person flow which is the last consumer of the `DocAuthBaseStep`.